src/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,81 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import NextAuth, { NextAuthOptions } from "next-auth";
+import { NextApiHandler } from "next-auth/internals/utils";
+import { toArray } from "../../../lib/utils";
+
+const tenantName = process.env.AZURE_B2C_TENANT_NAME;
+const tenantId = process.env.AZURE_B2C_TENANT_ID;
+const userFlow = process.env.AZURE_B2C_LOGIN_FLOW;
+const clientId = process.env.AZURE_B2C_CLIENT_ID;
+const clientSecret = process.env.AZURE_B2C_CLIENT_SECRET;
+
+const accessTokenUrl = `https://${tenantName}.b2clogin.com/${tenantName}.onmicrosoft.com/${userFlow}/oauth2/v2.0/token`;
+const requestTokenUrl = `https://${tenantName}.b2clogin.com/${tenantName}.onmicrosoft.com/${userFlow}/oauth2/v2.0/token`;
+const authorizationUrl = `https://${tenantName}.b2clogin.com/${tenantName}.onmicrosoft.com/${userFlow}/oauth2/v2.0/authorize?response_type=code+id_token&response_mode=form_post`;
+
+const options: NextAuthOptions = {
+  session: {
+    jwt: true,
+  },
+  jwt: {
+    encryption: true,
+    secret: process.env.JWT_SECRET,
```

Comment:
Ensures that JWT are encrypted within the browser, see docs: https://next-auth.js.org/configuration/options#jwt

The default behaviour is to sign the tokens (JWS), but not encrypt (JWE)